### PR TITLE
The player is functional on all pages 

### LINKED ISSUE #97: Insert all playlist Ids in script tags
The player requires a playlist ID in order to start playing the videos.

### PR DESCRIPTION
**Pull request by smarthutza**
The player displays the right videos on each of the pages. 
There is some commented out code in the player.js file, but I will need it in the future. Please leave it there. 

Fix #97

### CODE REVIEW

File: public/logic.js [hunk 1]
```diff
@@ -44,5 +44,6 @@ function changePage () {
 }
 
 document.querySelectorAll('figure.five-videos, figure.three-videos, figure.two-videos, button#confirm-survey-button, button#ok-submit-survey, button#pre-session-button, button#post-session-button').forEach((e, i) => {
+  console.log('I have reached this line')
```
Comment (des-des) [inline public/logic.js]: delete

File: public/player.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// 2. This code loads the IFrame Player API code asynchronously.
```
Comment (des-des) [inline public/player.js]: wrap this file in a iife

**Review by des-des — APPROVED** [review]

**Review by des-des — CHANGES_REQUESTED** [review]
ops see above

**Comment by lucyrose93** [comment]
@smarthutza you will need to merge this with master again when you address Eoin's feedback. 

Please try to make sure that it doesn't affect session storage and layout. To get an idea of how it should look, compare it with master at the moment, which has a nice layout :)

**Comment by lucyrose93** [comment]
I am closing this PR to replace it with an updated one that is merged with master

see https://github.com/lucyrose93/MakaTalk/pull/108

**Pull request closed without merging**